src/webots/nodes/utils/WbSupervisorUtilities.cpp

```diff
@@ -437,7 +438,7 @@ void WbSupervisorUtilities::handleMessage(QDataStream &stream) {
       return;
     }
     case C_SUPERVISOR_SIMULATION_RESET:
-      WbApplication::instance()->simulationReset();
+      mNeedToResetSimulation = true;  // WbApplication::instance()->simulationReset(false);
```

Comment:
What is the purpose of the commented instruction?
If you want to describe which action is called, maybe some additional text would be better.

Reply:
There is actually no purpose, I just forget to remove the comment.